src/main/java/step4/generator/AutoLottoGenerator.java

```diff
@@ -9,21 +11,22 @@
 import java.util.stream.IntStream;
 
 public class AutoLottoGenerator implements LottoGenerator {
-    private static final int LOTTO_PRICE = 1000;
     private static final int MIN_NUMBER = 1;
     private static final int MAX_NUMBER = 45;
 
     @Override
-    public List<Lotto> generate(int money) {
+    public Lottos generate(Money money) {
         List<Lotto> lottos = new ArrayList<>();
-        for(int i=0; i<money / LOTTO_PRICE; i++){
+        for (int i = 0; i < money.lottoCount(); i++) {
             lottos.add(makeLotto());
         }
-        return lottos;
+        return new Lottos(lottos);
     }
 
     public Lotto makeLotto() {
```

Comment:
`AutoLottoGenerator` 외에 사용하는 곳이 없어 보이네요. public이 아닌 private으로 바꾸면 어떨까요?

```suggestion
    private Lotto makeLotto() {
```